plugins/Quotes/class.quotes.plugin.php

```diff
@@ -302,6 +301,13 @@ public function PostController_BeforeCommentRender_Handler($Sender) {
    }
 
    protected function FormatQuote($Type, $ID, &$QuoteData, $Format = FALSE) {
+      // Temporarily disable Emoji parsing (prevent double-parsing to HTML)
+      $undoEmoji = false;
```

Comment:
Y'know I even thought about that afterward and decided not to and now I have no idea why.